_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -286,11 +286,11 @@ class MyPlanBody extends React.Component {
 						( 'inactive' !== this.props.getModuleOverride( 'videopress' ) ) && (
 							<div className="jp-landing__plan-features-card">
 								<h3 className="jp-landing__plan-features-title">{ __( 'Video Hosting' ) }</h3>
-								<p>{ __( 'Fast, optimized, ad-free, and unlimited video hosting for your site.' ) }</p>
+								<p>{ __( 'High-speed, high-definition video hosting that uses your server space efficiently, and comes with no third-party ads.' ) }</p>
```

Comment:
I am not sure I understand "uses your server space efficiently". VideoPress videos are not uploaded at all on the site owner's server, so it does not use the server at all.

Reply:
How does this sound?

> High-speed, high-definition video hosting with no third-party ads.

We should also change the card in Calypso, where this copy originally came from:

![image](https://user-images.githubusercontent.com/390760/49954468-75699b80-fef8-11e8-9dec-6b8b73afb547.png)

https://github.com/Automattic/wp-calypso/blob/59bdfeeb97eda4266ad39410cb0a074d2c88dbc8/client/blocks/product-purchase-features-list/jetpack-video.jsx#L22

Reply:
That works for me!

Reply:
Addressed in https://github.com/Automattic/wp-calypso/commit/17eaa1ae5eeba9343fc01178c5f04b5dce8a0e30

Also created a PR for Calypso in https://github.com/Automattic/wp-calypso/pull/29428